strawberryfields/ops.py

```diff
@@ -1808,8 +1808,8 @@ class S2gate(Gate):
             :class: defn
 
             .. math::
-                S_2(z) = \exp\left(z^* \a_1\a_2 -z \ad_1 \ad_2 \right) =
-                \exp\left(r (e^{-i\phi} \a_1\a_2 -e^{i\phi} \ad_1 \ad_2 \right)
+                S_2(z) = \exp\left(z \a^\dagger_1\a^\dagger_2 -z^* \a_1 \a_2 \right) =
```

Comment:
At the very top of this docstring it uses a and b, so we should at least be internally consistent as well. Can you update?

Reply:
I changed the docstring.